### PR TITLE
Adds definition for Pro 2 v.1 UL

### PR DESCRIPTION
**Pull request by tomcastleman**
Addresses #700 for Pro v.1 UL

### CODE REVIEW

**Comment by thecode** [comment]
@tomcastleman can you post output from `http://device_ip/shelly` ?

**Comment by tomcastleman** [comment]
@thecode sure:

```json
{
  "name":null,
  "id":"shellypro2-xxxxxxxxxxxx",
  "mac":"XXXXXXXXXXXX",
  "slot":1,
  "model":"SPSW-202XE12UL",
  "gen":2,
  "fw_id":"20241011-114450/1.4.4-g6d2a586",
  "ver":"1.4.4",
  "app":"Pro2",
  "auth_en":false,
  "auth_domain":null
}
```

File: aioshelly/const.py [hunk 1]
```diff
@@ -699,6 +700,13 @@ class ShellyDevice:
         gen=GEN2,
         supported=True,
     ),
+    MODEL_PRO_2_V3_UL: ShellyDevice(
+        model="SPSW-202XE12UL",
+        name="Shelly Pro 2",
```
Comment (thecode) [inline aioshelly/const.py:705]:
```suggestion
        name="Shelly Pro 2 UL",
```

I noticed that for the Plus we did specify in the name that the device is UL:
https://github.com/home-assistant-libs/aioshelly/blob/e60f9f0d009710042cf70d1527eedbe1096ce7aa/aioshelly/const.py#L537

**Review by thecode — APPROVED** [review]

**Comment by thecode** [comment]
Thanks @tomcastleman 👍

**Pull request merged**